jupyterhub_config-workspace.py

```diff
@@ -66,33 +75,7 @@
         {
             'name': 'data',
             'mountPath': '/opt/app-root',
-            'subPath': 'workspace'
-        }
-    ])
-
-    c.KubeSpawner.init_containers.extend([
-        {
-            'name': 'setup-volume',
-            'image': '%s' % c.KubeSpawner.image_spec,
-            'command': [
-                '/opt/app-root/bin/setup-volume.sh',
-                '/opt/app-root',
-                '/mnt/workspace'
-            ],
-            "resources": {
-                "limits": {
-                    "memory": os.environ.get('NOTEBOOK_MEMORY', '128Mi')
-                },
-                "requests": {
-                    "memory": os.environ.get('NOTEBOOK_MEMORY', '128Mi')
-                }
-            },
-            'volumeMounts': [
-                {
-                    'name': 'data',
-                    'mountPath': '/mnt'
-                }
-            ]
+            'subPath': 'private-workspace'
```

Comment:
Why is this private workspace?
Btw the pvc was never created in the Azure prod.